hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1554,9 +1554,9 @@
   <property>
     <name>hdds.datanode.replication.work.dir</name>
     <tag>DATANODE</tag>
-    <description>Temporary which is used during the container replication
-      betweeen datanodes. Should have enough space to store multiple container
-      (in compressed format), but doesn't require fast io access such as SSD.
+    <description>This configuration is deprecated. Default directory which is used
```

Comment:
This configuration is deprecated.  Temporary sub directory under each hdds.datanode.dir will be used during the container replication between datanodes.